src/breadcrumb-group/__tests__/full-dropdown.test.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { FullDropdown } from '../../../lib/components/breadcrumb-group/full-dropdown.js';
+import createWrapper, { ButtonDropdownWrapper } from '../../../lib/components/test-utils/dom';
+
+const items = [
+  {
+    text: 'Breadcrumb text',
+    href: 'http://amazon.com',
+  },
+  {
+    text: '',
+    href: '',
+  },
+  {
+    text: 'Last item',
+    href: '#',
+  },
+];
+describe('FullDropdown', () => {
+  let wrapper: ButtonDropdownWrapper;
+  beforeEach(() => {
+    const { container } = render(<FullDropdown items={items} onClick={jest.fn()} onFollow={jest.fn()} />);
```

Comment:
I think it would make sense to define the full-dropdown tests in context of the breadcrumb group: render the breadcrumb group with all items collapsed and then run the tests. This will ensure that the selectors to find the full dropdown from the button group root wrapper are covered, too.

Btw, should we add more test utils?

Reply:
yeah, I think that makes sense, i'll consolidate.

I don't think we need more test utils: this new dropdown is still findable via the existing `findDropdown` util, and items within the new and old dropdowns can be interacted with in the same way. I could _potentially_ see the need to check which mode the breadcrumbs overall is in, but didn't need that for any of the tests so far, we can always add in the future if requested.